examples/MyModule/step_pattern_builder.cpp

```diff
@@ -101,10 +101,7 @@ mynest::StepPatternBuilder::connect_()
 // This code is based on AllToAllBuilder, except that we step
 // by source_step_ and target_step_ except for stepping by 1.
 
-#pragma omp parallel
-  {
-    // get thread id
-    const int tid = nest::kernel().vp_manager.get_thread_id();
+  NEST_PARALLEL_FOR_THREAD(tid)
```

Comment:
👍 for extra curly braces, also to confuse basic text editors less. I'd prefer to stick with `tid` instead of `thread_id`, while the latter is clearer, the former is used way more throughout our codebase:
```bash
$ git grep " thread_id" . | wc -l
11
```

vs.

```bash
$ git grep " tid" . | wc -l
182
```

maybe `tid` -> `thread_id` should be left for another PR?